services/payments/tests/unit/test_rpc_payments.py

```diff
@@ -18,49 +20,69 @@
 from simcore_service_payments.api.rpc.routes import PAYMENTS_RPC_NAMESPACE
 
 pytest_simcore_core_services_selection = [
+    "postgres",
     "rabbit",
 ]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
 
 
 @pytest.fixture
 def app_environment(
     monkeypatch: pytest.MonkeyPatch,
     app_environment: EnvVarsDict,
     rabbit_env_vars_dict: EnvVarsDict,  # rabbitMQ settings from 'rabbit' service
+    postgres_env_vars_dict: EnvVarsDict,
+    wait_for_postgres_ready_and_db_migrated: None,
 ):
+    # set environs
     monkeypatch.delenv("PAYMENTS_RABBITMQ", raising=False)
+    monkeypatch.delenv("PAYMENTS_POSTGRES", raising=False)
```

Comment:
are you disabling rabbitmq and postgres here? why not use your fixtures for this instead? that is confusing.